flocker_bb/builders/flocker.py

```diff
@@ -534,6 +625,16 @@ def getBuilders(slavenames):
                       category='flocker',
                       factory=makeAdminFactory(),
                       nextSlave=idleSlave),
+        BuilderConfig(name='flocker-homebrew-creation',
+                      slavenames=slavenames['fedora'],
+                      category='flocker',
+                      factory=makeHomebrewRecipeCreationFactory(),
+                      nextSlave=idleSlave),
+        BuilderConfig(name='flocker-homebrew-test',
+                      slavenames=slavenames['osx-10'],  # XXX Slave name?
```

Comment:
This should be `'osx'`. You can look above to see the other builder running on the slave.